standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -4055,7 +4055,6 @@ public boolean equals(Object obj) {
   private List<Partition> add_partitions_core(final RawStore ms, String catName,
                                               String dbName, String tblName, List<Partition> parts, final boolean ifNotExists)
       throws TException {
-    logAndAudit("add_partitions");
```

Comment:
I would suggest not removing this line. This is helpful in debugging/audit logs.

Reply:
The audit log is recorded in public `add_partitions_xxx` methods, so I think we do not need repeat it in this common function.